content/docs/studio/get-started.md

```diff
@@ -1,31 +1,38 @@
 # Get Started with DVC Studio
 
 Here, we will walk you through a tutorial to use DVC Studio for collaboration on
-your ML projects. You will need access to a Github, Gitlab or Bitbucket account
+your ML projects. You will need access to a GitHub, GitLab or Bitbucket account
```

Comment:
in the "DVC Studio creates views" view is not defined yet. Better to use some human readable terms - e.g. table with experiments? etc

Reply:
Will add a link from 'views' to the 'What is a view?' section on that page. Works?